### ISSUE TITLE
Fix reference to PEP8 in config setting

### ISSUE THREAD

**Issue by nikolay** [AtomLinter/linter-flake8#199]
https://twitter.com/gvanrossum/status/737438918799282178

**Comment by Arcanemagus** [comment]
I think you meant to file this on `linter-pep8`, in which case it's a duplicate of https://github.com/AtomLinter/linter-pep8/issues/55.

**Comment by nikolay** [comment]
@Arcanemagus No, here as flake8 depends on PEP8, and maybe even the branding will change.

**Comment by Arcanemagus** [comment]
`flake8` depending on `pep8`/`pycodestyle` has _nothing_ to do with this package.

This is simply a wrapper around `flake8`, how it integrates `pycodestyle` is completely up to it.

**Comment by Arcanemagus** [comment]
Looks like [this merge request](https://gitlab.com/pycqa/flake8/merge_requests/60) is what you should follow for tracking that.
